docroot/modules/custom/tfa_unblock/tfa_unblock.module

```diff
@@ -0,0 +1,6 @@
+<?php
```

Comment:
A .module is not needed as of Drupal 8. Lets add one when we need it.

Reply:
Ok, will remove now